docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,284 @@
+# Create Your First Tizen .NET NUI Application
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Natural User Interface (NUI) application development process. With the instructions, you can create and run a basic NUI application, which displays some text on the screen with no user interaction.
+
+   > **Note**
+   >
+   > .NET NUI applications are supported since Tizen 4.0
```

Comment:
Add a period **.** at the end of the sentence.

 .NET NUI applications are supported since Tizen 4.0.